src/tiled/command.cpp

```diff
@@ -209,12 +216,20 @@ CommandProcess::CommandProcess(const Command &command, bool inTerminal)
 
     connect(this, SIGNAL(finished(int)), SLOT(deleteLater()));
 
+    if (showOutput)
+        connect(this, &QProcess::readyReadStandardOutput, this, &CommandProcess::consoleOutput);
```

Comment:
You'll want to also connect to `readyReadStandardError` or use [`setProcessChannelMode`](http://doc.qt.io/qt-5/qprocess.html#setProcessChannelMode) to merge the standard error channel into the standard out channel.

Of course, it would be nice to report the error channel as appropriate to the `LoggingInterface`, so that the Debug Console will show errors in red.